bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -376,7 +416,10 @@ impl OmniConnector {
             self.token_locker_id()?.to_string(),
             "sign_transfer".to_string(),
             serde_json::json!({
-                "nonce": origin_nonce.to_string(),
+                "transfer_id": {
+                    "origin_chain": "Near", // TODO: provide transfer_id instead of only nonce
```

Comment:
Providing a chain by string feels very insecure. Can we use `ChainKind` enum here?

Reply:
Done